src/main/jbake/content/documentation/bundles/org-apache-sling-junit-bundles.md

```diff
@@ -27,6 +27,42 @@ the test class names, like for example:
 
     Sling-Test-Regexp=com.example.*ServerSideTest
 
+### JUnit 4 Support
+All that is required is the installation of the the Apache Sling JUnit Core bundle. The bundle exports the packages `junit.*`, `org.junit.*` and additionally `org.hamcrest.*`. Note however that the `org.junit.platform.*` packages, which contain the generic testing platform developed for JUnit 5, are NOT exported.  
+
+### JUnit 5 Support (since version 1.1.0)
+JUnit 5 support requires installation of a number of bundles provided by the JUnit team. The Apache Sling JUnit Core bundle has optional imports for some of the packages exported by these bundles and will automatically switch to JUnit 5 support if these are available (a bundle restart is required).
```

Comment:
"automatically switch to JUnit 5" might imply that JUnit 4 tests stop working when that switch is done.

If that's not the case I would say "automatically activate JUnit 5 support" instead.

Reply:
I'm currently rewording the JUnit 5 Support section. I agree the wording is confusing here, so I'll try to improve it.

To clarify: once the bundle switches to JUnit 5 support, it runs tests using the JUnit Platform. This allows running JUnit 5 tests  using the JUnit Jupiter engine and JUnit 3+4 tests using the JUnit Vintage engine. And if both engines are available, both test styles are supported side-by-side.